frontend/common/src/components/Hero/Hero.tsx

```diff
@@ -0,0 +1,54 @@
+import React from "react";
+
+import Heading from "../Heading";
+import Breadcrumbs, { type BreadcrumbsProps } from "../Breadcrumbs/v2";
+
+import "./hero.css";
+
+interface HeroProps {
+  imgPath: string;
+  title: React.ReactNode;
+  subtitle?: React.ReactNode;
+  crumbs?: BreadcrumbsProps["crumbs"];
+}
+
+const Hero = ({ imgPath, title, subtitle, crumbs }: HeroProps) => (
```

Comment:
Could we get a story for this new component?

Reply:
Added in [add stories for breadcrumbs + hero](https://github.com/GCTC-NTGC/gc-digital-talent/pull/4293/commits/310aca756461a738206a8b53f7725df6b56eb1be)